packages/plugin-e2e/src/models/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { Page } from '@playwright/test';
+
+export const radioButtonSetChecked = async (
```

Comment:
Planning to expose some kind of UI helpers through the base page, but will do that in a different [pr](https://github.com/grafana/plugin-tools/pull/703).